### DISCUSSION TITLE
Adding capability

### DISCUSSION THREAD

**Discussion by batutahibnu17** [projectdiscovery/nuclei-templates#11114] [Q&A] [answered]
Hi.
How to add capabilities, for example I want to add a hash of a .ico to a favicon-detect template that I am not the template creator (owner) of?

**Comment by rxerium** [comment, marked as answer]
Hey @batutahibnu17, anyone can contribute regardless of the `author`, feel free to create a PR with your changes. :)

**Reply by batutahibnu17** [reply]
Thanks mate

**Comment by batutahibnu17** [comment]
ok
